lib/fizzy/execute.cpp

```diff
@@ -565,7 +565,8 @@ std::unique_ptr<Instance> instantiate(Module module,
 
     // We need to create instance before filling table,
     // because table functions will capture the pointer to instance.
-    // FIXME: clang-tidy warns about potential memory leak for moving memory (which is in fact
+    //
+    // NOTE: clang-tidy warns about potential memory leak for moving memory (which is in fact
```

Comment:
@chfast wanted to deal with it at some point, that's why it was `FIXME`

Reply:
So change it to `TODO`? I don't think it is a bug in our code.

Reply:
Yes, I'd make it `TODO`

Reply:
Fixed.